static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -9,28 +9,26 @@ import type {
   IntegrationProvider,
   OrganizationIntegration,
 } from 'sentry/types/integrations';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {getIntegrationFeatureGate} from 'sentry/utils/integrationUtil';
 import {useApiQueries, useApiQuery} from 'sentry/utils/queryClient';
 import useOrganization from 'sentry/utils/useOrganization';
 import MessagingIntegrationModal from 'sentry/views/alerts/rules/issue/messagingIntegrationModal';
 
 export enum MessagingIntegrationAnalyticsView {
-  ALERT_RULE_CREATION = 'alert_rule_creation',
-  PROJECT_CREATION = 'project_creation',
+  ALERT_RULE_CREATION = 'alert_rule_creation_messaging_integration_onboarding',
+  PROJECT_CREATION = 'project_creation_messaging_integration_onboarding',
+  DEFAULT = 'messaging_integration_onboarding',
```

Comment:
is this being used anywhere? I was curious if it was being used as the default for naming, then i didn't see it anywhere else in the PR 😅

Reply:
oh oops i added the default as a janky fix for something but then i didn't end up needing it 😅  thanks for catching this!